app/bt/Home/index.tsx

```diff
@@ -45,7 +45,7 @@ const styles = StyleSheet.create({
   iconContainer: {
     position: 'absolute',
     left: -(Layout.screenWidth / 2),
-    bottom: -(Layout.screenHeight / 2),
+    bottom: -(Layout.screenHeight / 3),
```

Comment:
This whole icon as a background image is a bit wonky.  What do you think about using an actual background image for the concentric circles?

Reply:
I agree that it's wonky - the only benefit is that it ensures alignment with the bg rings and the icon centered on them. I think we can prevent any issues if we separate the rings in a bg image and position that and the icon absolute to the top? It's also implemented like this throughout, do you think that change should be tackled in a separate PR (or separate commit in this PR) so it can be rolled back individually?